app/validators/invite_validator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class InviteValidator < ActiveModel::Validator
+  LIMIT = (ENV['MAX_ACTIVE_INVITES'] || 5).to_i
+  DAILY_LIMIT = (ENV['MAX_DAILY_INVITES'] || 20).to_i
+  MINIMUM_ACCOUNT_AGE = 7.days
+
+  def validate(invite)
+    return if invite.user.can?(:manage_invites)
```

Comment:
I'm not too happy with checking if the user can manage invites here.
Alternatively,  limits could be bypassed if a user has a role as usually these are given to trusted users.
I decided on checking for `manage_invites` since it's the safest option.
Note that moderators don't have this permission by default.

Reply:
Actually, I could re-use the `change_max_use` permission.
Probably change the name to `unrestricted_invites` and give it to moderators and up.